style.css

```diff
@@ -80,3 +80,515 @@ body {
   gap: 20px;
   padding: 0;
 }
+
+.card-works {
+  border: 1 px solid #fff;
+  margin-top: 142px;
+  position: absolute;
+  width: 100%;
+  top: 950px;
+  display: contents;
```

Comment:
- [ ] Please change the display to grid. It is a requirement to use grid for this section of the document.